GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -275,6 +294,70 @@ private void loadTagInfo(string revision)
             this.InvokeAsync(updateText);
         }
 
+
+        private void loadCommitSignInfo(GitRevision revision)
+        {
+            /* Good commit signature */
+            if (revision.SignInfo == "G")
```

Comment:
From [git log pretty format page](https://git-scm.com/docs/pretty-formats):
> '%G?': show "G" for a good (valid) signature, "B" for a bad signature, "U" for a good signature with unknown validity, "X" for a good signature that has expired, "Y" for a good signature made by an expired key, "R" for a good signature made by a revoked key, "E" if the signature cannot be checked (e.g. missing key) and "N" for no signature
>
this "G" is the answer for valid signature, so it never will be "g".
I can define that character as static readonly if you prefer.